Framework/CurveFitting/src/Algorithms/Fit1D.cpp

```diff
@@ -779,11 +778,10 @@ void Fit1D::exec() {
 FitData::FitData(Fit1D *fit, const std::string &fixed)
     : n(0), X(nullptr), Y(nullptr), sigmaData(nullptr), fit1D(fit),
       forSimplexLSwrap(nullptr), parameters(nullptr) {
-  typedef boost::tokenizer<boost::char_separator<char>> tokenizer;
-  boost::char_separator<char> sep(",");
-  boost::tokenizer<boost::char_separator<char>> names(fixed, sep);
+  Mantid::Kernel::StringTokenizer namesStrTok(
+      fixed, ",", Mantid::Kernel::StringTokenizer::TOK_TRIM);
   active.insert(active.begin(), fit1D->m_parameterNames.size(), true);
-  for (tokenizer::iterator it = names.begin(); it != names.end(); ++it) {
+  for (auto it = namesStrTok.begin(); it != namesStrTok.end(); ++it) {
```

Comment:
I'm not sure about removing those next three lines as they were there before and I haven't dealt with this code before so I can't say if they're necessary or not. If anyone else could explain if they are needed or not that would be good.